Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -110,6 +113,30 @@ export const getWeeksInMonth = function (year) {
 
 export const getCurrentDay = function (dummy) {
   var date = new Date();
-  return { utcDate: date.toISOString(), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
+  // var startDate = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+  return { day: getISODate(date), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
            isInRange : false, isStart: false , isEnd: false }
 }
+
+export const compareDates = function (day1){
+  return function (day2) {
+    if (day1 == "" || day2 == "") return "INV";
+    let dateOne = new Date(day1).toISOString();
```

Comment:
Add a try catch block for the case, where the date conversion using new Date() fails.